XenModel/XenAPI-Extensions/Host.cs

```diff
@@ -548,6 +548,16 @@ public static bool RestrictLivePatching(Host h)
             return h._RestrictLivePatching;
         }
 
+        private bool _RestrictIGMPSnooping

```

Comment:
We recently converted all extension get properties of the API classes to methods, see for example here:
https://github.com/xenserver/xenadmin/pull/1757/commits/ae22560ce86f6a739e07807a6b74a321dedea7e1#diff-9249bdd29e594ac8a4520e24be0254d9L218
Please remove the `_RestrictIGMPSnooping`  property and move the code inside the `RestrictIGMPSnooping(Host h)` method.

Reply:
Updated. Thanks.